fix(ddx): change domain ownership (#2531)
scaleway/docs-content: dedibox-network/domains/how-to/change-ownership.mdx

content:
  h1: How to change the ownership of a domain name
  paragraph: This page explains how to change the ownership of a domain name
dates:
  validation: 2023-12-14
  posted: 2021-05-26
---

> You must always keep the contact details relating to the owner of your domain name accurate and up to date. This can be done by [following the instructions here](/dedibox-network/domains/how-to/file-contact-info). However, if you wish to change details pertaining to the identity of the owner (full name(s) of a natural person or corporate name for a legal entity) this must be done by the change of ownership process described below.

Reviewer: ```suggestion
You must always keep the contact details relating to the owner of your domain name accurate and up to date. This can be done by [following the instructions here](/dedibox-network/domains/how-to/file-contact-info). However, if you wish to change details pertaining to the identity of the owner (full name(s)) of a natural person or corporate name for a legal entity) this must be done by the change of ownership process described below.
```

Reviewer 2: Already a closing bracket after "legal entity"

> You must always keep the contact details relating to the owner of your domain name accurate and up to date. This can be done by [following the instructions here](/dedibox-network/domains/how-to/file-contact-info). However, if you wish to change details pertaining to the identity of the owner (full name(s) of a natural person or corporate name for a legal entity) this must be done by the change of ownership process described below.

Reviewer 2: ```suggestion
You must always keep the contact details relating to the owner of your domain name accurate and up to date. This can be done by [following the instructions here](/dedibox-network/domains/how-to/file-contact-info). However, if you wish to change details pertaining to the identity of the owner (full name(s) of a natural person or corporate name for a legal entity), this must be done by the change of ownership process described below.
```

5. Ensure the **ANTI SPAM** feature is set to **Inactive**.
6. If you have made changes, scroll to the bottom of the page and click **Update handle data**.

## How to change the ownership of a domain

1. From the console, click **Domain** at the top right of the screen.
2. Click **Trades**. Trading is the procedure that allows you to change the ownership of a domain.
    <Lightbox src="scaleway-trade-domain.webp" alt="" />
3. Click **Initiate a domain trade**.
4. Select the relevant domain name and click **Order**.
5. Select the account of the new owner, searching for it via the account login if necessary, and click **Confirm**.
  <Lightbox src="scaleway-trade-domain-2.webp" alt="" />
6. Create a handle for the new owner, containing all their contact details, and click **Confirm**.
7. Confirm the ownership trade and pay the administrative fee.
8. For the extensions `.fr`, `.eu` and `.be`, emails will be sent to both the old and new owners of the domain. Both parties are required to validate the trade by clicking the enclosed link. Without this action, the change of ownership will not be validated. 
  <Message type="important">

>     The link in the email is valid for one month. After this it expires, and to get a new validation link you will need to restart the change of ownership process. A new administrative fee will be applied.

Reviewer: ```suggestion
    Upon receipt of the email containing the validation link, you have one month to use it. To get a new validation link you will need to restart the change of ownership process. A new administrative fee will be applied.
```
